test/testsymboldatabase.cpp

```diff
@@ -2285,6 +2287,14 @@ class TestSymbolDatabase: public TestFixture {
         ASSERT_EQUALS(true, function && function->isImplicitlyVirtual(false));
     }
 
+    void functionStatic() {
+        GET_SYMBOL_DB("static void fs() {  }");
+        (void)db;
+        const Function *func = db->scopeList.back().function;
+        ASSERT_EQUALS(true, func != nullptr);
+        ASSERT_EQUALS(true, func ? func->isStatic() : false);
```

Comment:
Nowadays `ASSERT` should stop if the condition is false. So you don't have to write the asserts like this. You can just write this instead:

    ASSERT(func);
    ASSERT(func->isStatic());

There is old test code that should be cleaned up. You probably saw that.

My plan is to add `EXPECT` and `EXPECT_EQUALS` later that will not abort. Feel free to do that sometime if you want.. but not part of this PR of course.

Reply:
Thanks, I fixed it. I'll look at this next time.